### ISSUE TITLE
Maybe use reqwest default-tls?

### ISSUE THREAD

**Issue by ikatson** [ducaale/ht#12]
Just tried running ```ht https://www.google.com```, and got an error

```thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Body, source: hyper::Error(Body, Error { kind: Proto(PROTOCOL_ERROR) }) }'```

Also tried on another website and got
```
 Custom { kind: InvalidData, error: PeerMisbehavedError("received unadvertised sig scheme RSA_PKCS1_SHA1") }
```

After recompiling with reqwests's default feature "deafult-tls", both websites worked fine.

**Comment by ducaale** [comment]
As @Till--H [mentioned](https://github.com/ducaale/ht/issues/13#issuecomment-774442764), the problem was due to ht setting the `HOST` header which is not allowed for HTTP 2 requests. As a workaround, you can unset the host header `ht https://www.google.com host:` or you can build ht from the master branch which contains a fix.

**Comment by ducaale** [comment]
This is now fixed as part of the ht v0.5.0 release 🎉
